pkg/storage/stores/indexshipper/compactor/deletion/delete_requests_manager.go

```diff
@@ -270,6 +270,7 @@ func (d *DeleteRequestsManager) Expired(ref retention.ChunkEntry, _ model.Time)
 			level.Info(util_log.Logger).Log(
 				"msg", "no chunks to retain: the whole chunk is deleted",
 				"delete_request_id", deleteRequest.RequestID,
+				"sequence num", deleteRequest.SequenceNum,
```

Comment:
I'd suggest making these snake_case to match the rest

```suggestion
				"sequence_num", deleteRequest.SequenceNum,
```

Reply:
Good catch.